src/Servers/Kestrel/Core/src/Internal/Http/HttpProtocol.Generated.cs

```diff
@@ -11,6 +11,8 @@
 using Microsoft.AspNetCore.Http.Features.Authentication;
 using Microsoft.AspNetCore.Server.Kestrel.Core.Features;
 
+#pragma warning disable CA2252 // WebTransport is a preview feature
```

Comment:
I think this is what Daniel was telling me for a reason he didn't set the preview attribute higher, it's a bit viral. But it's better from a "we can break this in the future" perspective.

By suppressing it normal apps won't see the warning until they touch `IHttpWebTransportFeature` or `IWebTransportSession` right?

Reply:
> By suppressing it normal apps won't see the warning until they touch `IHttpWebTransportFeature` or `IWebTransportSession` right?

Correct. Without marking the entire interface as preview, we'd have to consider whether were breaking both callers and implementers of this interface.

Reply:
Yeah, this is exactly why we didn't do it this way earlier. Your fix is correct, but annoying to work with 😁